lib/streamlit/elements/widgets/chat.py

```diff
@@ -67,11 +69,14 @@ def _process_avatar_input(
 
     if avatar is None:
         return AvatarType.ICON, ""
-    elif isinstance(avatar, str) and avatar in [
-        PresetNames.USER,
-        PresetNames.ASSISTANT,
-    ]:
-        return AvatarType.ICON, avatar
+    elif isinstance(avatar, str) and avatar in set(item.value for item in PresetNames):
```

Comment:
nit: `set(item.value for item in PresetNames)` is written here while the equivalent `{item.value for item in PresetNames}` is instead on L193 below.

Would be preferable to be consistent (and I personally prefer the set comprehension over using the `set` constructor, but no strong opinion about which of the two to use)

Reply:
Updated to use set comprehension here as well 👍